ffs/rpc/service.go

```diff
@@ -530,6 +530,33 @@ func (s *Service) AddToHot(srv FFS_AddToHotServer) error {
 	return srv.SendAndClose(&AddToHotReply{Cid: c.String()})
 }
 
+// ActiveMiners returns information about the miners the specified FFS instance has deals with
+func (s *Service) ActiveMiners(ctx context.Context, req *ActiveMinersRequest) (*ActiveMinersReply, error) {
```

Comment:
Here's an example of the kind of method that is going to be implemented to the ui requirements. It used the general query method to get all the `CidInfo`s for the ffs instance, then groups stored cids by miner.

Reply:
If we do go down this path of implementing some things for the web ui only in RPC layer, we could even create a separate rpc package/service just for this category of things. This might help keep straight what is directly exporting the ffs api vs more custom client driven requirements. Overkill?

Reply:
> If we do go down this path of implementing some things for the web ui only in RPC layer, we could even create a separate rpc package/service just for this category of things. This might help keep straight what is directly exporting the ffs api vs more custom client driven requirements. Overkill?

Totally agree, good idea to avoid exploding ffs with custom stuff.